validator/rules/rule_config.go

```diff
@@ -0,0 +1,102 @@
+package rules
+
+import (
+	"fmt"
+	"github.com/aerogear/charmil/validator"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+type Rule interface {
+	Validate(command *cobra.Command, ruleConfig *RuleConfig) []validator.ValidationError
+}
+
+// RuleConfig is the struct that stores
+// configuration of rules
+type RuleConfig struct {
+	Verbose bool
+	Length
+	MustExist
+}
+
+func NewRuleConfig() RuleConfig {
+	return RuleConfig{
+		Verbose: false,
+		Length: Length{
+			Limits: map[string]Limit{
+				"Use":     {Min: 2},
+				"Short":   {Min: 15},
+				"Long":    {Min: 50},
+				"Example": {Min: 50},
+			},
+		},
+		MustExist: MustExist{
+			Fields: []string{"Use", "Short", "Long", "Example"},
+		},
+	}
```

Comment:
@jackdelahunt Actually in my current PR #105, things are changed. Mainly RuleConfig struct.
Based upon #110

Reply:
not able to add you as reviewer there. Want your review #105

Reply:
Agh!! I didn't see this pr you do alot of what I just did. I'll close this an maybe look into it again when your pr is merged